### ISSUE TITLE
Can't change Download ToolTip

### ISSUE THREAD

**Issue by warpor** [apexcharts/Blazor-ApexCharts#415]
The `LocaleToolbar` class has a `Download` attribute which is for changing tootlip, but changing this parameter has no effect on tooltip(It's always "Menu"). Other attributes (for example `ZoomIn`) work correctly. I assume that the error associated with the Js-script. In it `title` is assigned the value of attribute `menu`, not `download`, and in `LocalToolBar` there is no attribute `Menu`.

code from `apexchart.esm.js`

```
el: this.elMenuIcon,
icon: "string" == typeof this.t.download ? this.t.download : '<svg xmlns="http://www.w3.org/2000/svg" width="24" height="24" viewBox="0 0 24 24"><path fill="none" d="M0 0h24v24H0V0z"/><path d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"/></svg>',
title: this.localeValues.menu,
class: "apexcharts-menu-icon"
                
```
                
![photo1708933001](https://github.com/apexcharts/Blazor-ApexCharts/assets/31928576/acce7bc4-71fb-49b7-b426-a67252409e67)
![photo1708932873](https://github.com/apexcharts/Blazor-ApexCharts/assets/31928576/727fd168-38f9-485d-a2cc-2b73e6314c78)

**Comment by joadan** [comment]
Released in version 3.0

**Comment by joadan** [comment]
Thank you!
